src/server/api/routes/users/v2/follow.js

```diff
@@ -92,18 +100,28 @@ router.delete(
         res.status(400).send('INVALID_CREATOR_ADDRESS');
         return;
       }
-      await walletUserCollection.doc(user).update({
-        followees: FieldValue.arrayRemove(creator),
+      await db.runTransaction(async t => {
+        const promises = [
+          t.update(walletUserCollection.doc(user), {
```

Comment:
https://cloud.google.com/nodejs/docs/reference/firestore/latest/firestore/transaction#_google_cloud_firestore_Transaction_update_member_1_
transation .update is not a promise, so we cannot put this here
Need to put all .get() before .update()